### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ruirlliu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fixed typo

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apimachinery/pkg/api/validation/path/name_test.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func TestValidatePathSegmentName(t *testing.T) {
 		},
 		// Make sure non-ascii characters are tolerated
```
Comment (caesarxuchao) [inline staging/src/k8s.io/apimachinery/pkg/api/validation/path/name_test.go]: Reading this comment, it seems to be intentional to use non-ascii characters.

Comment (louiseveryd) [reply staging/src/k8s.io/apimachinery/pkg/api/validation/path/name_test.go]: Got it ，i will close  this pr

**Pull request closed without merging**